test/hotspot/jtreg/vmTestbase/nsk/jdi/Event/request/request001.java

```diff
@@ -554,11 +550,14 @@ private void testRun()
                 } else {
                     log3("ERROR: else clause in detecting type of event1");
                     testExitCode = FAILED;
+                    throw new JDITestRuntimeException("** unexpected event **");
```

Comment:
It is better to print the event type (or, at least, something) in this case.

Reply:
Just add ` + event`.